styles/github/catppuccin.user.css

```diff
@@ -1,13 +1,13 @@
 /* ==UserStyle==
```

Comment:
Got a merge mistake here, you can `git checkout main styles/github/catppuccin.user.css` & commit to reset.